exampleSite/content/talk/example-talk.md

```diff
@@ -0,0 +1,13 @@
++++
```

Comment:
Add something like `date = "2016-12-22T00:00:00"` since `date` is a required field. Note that `date` does not need to be added to archtypes because Hugo automatically requires `date` for all archtypes.